### PR TITLE
[Term Entry] D3 Interactivity .ease() added

### LINKED ISSUE #4018: [Term Entry] D3 Interactivity .ease()
### Reason/inspiration (optional)

We would like a new term entry on the `.ease()` method for [D3](https://www.codecademy.com/resources/docs/d3): Interactivity. The entry should go in a new file under `docs/content/d3/concepts/interactivity/terms/ease/ease.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by antriksh-9**
### Description
I was assigned this [issue](https://github.com/Codecademy/docs/issues/4018).
Created Codecademy Docs term entry on the .ease() method for D3.
Closes #4018.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/d3/concepts/interactivity/terms/ease/ease.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  - 'Methods'
+  - 'Animations'
+  - 'Selection'
+  - 'Transitions'
+CatalogContent:
+  - 'learn-d3'
+  - 'paths/web-development'
+---
+
+The **`.ease()`** method is specific to the D3.js library, which is primarily used for data visualization on the web. It allows you to control the timing and pacing of transitions in animations.
```
Comment (ishg-153) [inline content/d3/concepts/interactivity/terms/ease/ease.md:17]:
```suggestion
The **`.ease()`** method is specific to the D3.js library, this is primarily used to add an easing effect to a transition. It allows to control the timing and pacing of transitions in animations.
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@antriksh-9 I have suggested this one change for now. Please make it, I'll let you know if there's more.

**Comment by antriksh-9** [comment]
Hey @ishg-153 I have made the proposed changes.

File: content/d3/concepts/interactivity/terms/ease/ease.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+```pseudo
+selection.transition()
+    .duration(duration)
+    .ease(easingFunction);
+```
+
+- `selection`: The selection of DOM elements to which transition will be applied.
+- `duration`: The time duration of the transition in milliseconds.
+- `easingFunction`: It is the easing function.
```
Comment (ishg-153) [inline content/d3/concepts/interactivity/terms/ease/ease.md:29]: You should mention the different types of easing functions below this.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@antriksh-9 I have suggested one more change, please make it.

**Comment by antriksh-9** [comment]
Hey @ishg-153 I have made the necessary changes.

**Review by ishg-153 — APPROVED** [review]
@antriksh-9 Great! Ready for a second review!

File: content/d3/concepts/interactivity/terms/ease/ease.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            .attr("cx", 200)
+            .attr("cy", 40);
+    </script>
+
+</body>
+</html>
+```
+
+In this example, the `.ease(d3.easeLinear)` specifies that the transition should use a linear easing function. It provides a constant speed to the motion of the circle throughout the transition, resulting in a straight-line movement from the initial position to the final position.
+
```
Comment (avdhoottt) [inline content/d3/concepts/interactivity/terms/ease/ease.md:77]: Add a line describing that the following is the output of the given code.

File: content/d3/concepts/interactivity/terms/ease/ease.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- `d3.easeQuad`: Quadratic easing.
+- `d3.easeBounce`: Bouncing easing.
+- `d3.easeElastic`: Elastic easing with customizable amplitude and period.
+- `d3.easeCircle`: Circular easing, creating a circular motion.
+- `d3.easeBack`: Back easing with customizable overshoot.
+
+## Example
+
+Consider an HTML document with an SVG element containing a circle with the specified ID `myCircle`.
+Here's a simple HTML structure with D3.js code to create the transition animation:
```
Comment (avdhoottt) [inline content/d3/concepts/interactivity/terms/ease/ease.md:45]:
```suggestion
Consider an HTML document with an SVG element containing a circle with the specified ID `myCircle`. Here's a simple HTML structure with D3.js code to create the transition animation:

```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@antriksh-9 Great work! Some minor changes from my side.

**Comment by antriksh-9** [comment]
Hey @avdhoottt I have made the necessary changes.

**Review by avdhoottt — APPROVED** [review]
Ready to get merged.

**Pull request merged**